src/UI/UIController.java

```diff
@@ -35,5 +39,48 @@ public static boolean createUser(String username, String password) {
 	public static boolean loginUser(String username, String password) {
 		return User.loginThroughDatabase(databaseConnection, username, password);		
 	}
+	public static void setUserForSession(String username) {
+		Dao<UserEntity, String> userDao;
+		try {
+			userDao = DaoManager.createDao(databaseConnection, UserEntity.class);
+		
+		UserEntity userEntity = userDao.queryForId(username);
+		safeStoreUser = new User(userEntity);
+		UserSignInWindow.closeWindow();
+		FrontEnd.launchWindow();
+		}catch (SQLException e) {
+			// TODO Auto-generated catch block
+			e.printStackTrace();
+		}
+		
+	}
+	public static WebsiteAccount getWebsiteAccountInfo(String nickname) {
+			WebsiteAccount website = null;
+			try {
+				 website = WebsiteAccount.getWebsiteAccountFromNickname(databaseConnection, nickname, safeStoreUser);
+			} catch (Exception e) {
+				
+			}
+			return website;
+	}
+	public static boolean addWebsiteAccount(String nickname, String username, String password) {
+		WebsiteAccount websiteAccount = new WebsiteAccount(safeStoreUser, nickname, username, password);
+		return websiteAccount.addWebsiteAccount(databaseConnection);
+	}
+
+	public static boolean modifyWebsiteAccount(String currentUsername, String newNickname, String newLogin, String newPassword) {
```

Comment:
```suggestion
	public static boolean modifyWebsiteAccount(String currentNickname, String newNickname, String newLogin, String newPassword) {
``` it's current nickname right? otherwise how are you supposed to find the website to modify when the primary key is nickname